isodatetime/parser_spec.py

```diff
@@ -285,7 +285,13 @@ def __str__(self):
 
 
 def get_date_translate_info(num_expanded_year_digits=2):
-    expanded_year_digit_regex = "\d" * num_expanded_year_digits
+    """Return list of 4-element tuples with date translate information.
+
+    Each tuple is (
+        basic regex, regex with capture group, format substitution,
+        property name)
+    """
```

Comment:
Would be good if we could stick to a standard docstring format e.g: http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

I.E. something like...

```
returns:
    list: List of four element tuples with the following components.
        - basic regex (str) - Some description text
        - regex with capture group (str) - ...
        ...
```